bundle/ios/GutenbergNativeTranslations.swift

```diff
@@ -95,7 +96,6 @@ private func dummy() {
     _ = NSLocalizedString("Cut block", comment: "")
     _ = NSLocalizedString("Describe the purpose of the image. Leave empty if the image is purely decorative. ", comment: "")
     _ = NSLocalizedString("Dismiss", comment: "")
-    _ = NSLocalizedString("Display post date", comment: "")
```

Comment:
- Removal
- [Code reference](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/latest-posts/edit.js#L257:L257)

This string is already translated as it's used in the web version, so we don't need to include it here.